# Get an arm pose based on its name
string name
---
ArmPose pose
